keyboards/40percentclub/25/keymaps/macro/config.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2018 REPLACE_WITH_YOUR_NAME
```

Comment:
```suggestion
/* Copyright 2018 zvecr
```

or whatever you want your name to be.